spec/openapi3.yaml

```diff
@@ -3165,6 +3253,87 @@ components:
       allOf:
         - $ref: "#/components/schemas/EmbeddedTransactionDTO"
         - $ref: "#/components/schemas/MosaicGlobalRestrictionTransactionBodyDTO"
+    MosaicGlobalRestrictionRestrictionsItemDTO:
+      type: object
+      required:
+        - referenceMosaicId
+        - restrictionValue
+        - restrictionType
+      properties:
+        referenceMosaicId:
+          $ref: "#/components/schemas/MosaicId"
+        restrictionValue:
+          type: string
+          description: Restriction value.
+        restrictionType:
+          type: integer
+    MosaicGlobalRestrictionRestrictionsDTO:
+      type: object
+      required:
+        - key
+        - restriction
+      properties:
+        key:
+          type: string
+          description: Restriction key.
+          example: 0DC67FBE1CAD29E3
+        restriction:
+          type: array
+          items:
+            $ref: "#/components/schemas/MosaicGlobalRestrictionRestrictionsItemDTO"
+    MosaicGlobalRestrictionDTO:
+      type: object
+      required:
+        - compositeHash
+        - entryType
+        - mosaicId
+        - restrictions
+      properties:
+        compositeHash:
+          $ref: "#/components/schemas/Hash256"
+        entryType:
```

Comment:
So this is actually the restrictionType enum 0: address, 1: global. Should a enum be created for this?